src/serverExplorer.ts

```diff
@@ -37,6 +37,32 @@ export class ServersViewTreeDataProvider implements TreeDataProvider<Protocol.Se
         this.refresh();
     }
 
+    restartServer(client, timeout: number = 6000) {
```

Comment:
`client` parameter is not required here because serverExplorer instance gets client's instance in constructor. 
`timeout` is not needed ether, because it is part of client start/stop server methods.